serverless/lambda/src/main/java/uk/gov/di/lambdas/ClientInfoHandler.java

```diff
@@ -79,6 +79,10 @@ public APIGatewayProxyResponseEvent handleRequest(
             Map<String, List<String>> authRequest = clientSession.get().getAuthRequestParams();
 
             String clientID = AuthenticationRequest.parse(authRequest).getClientID().getValue();
+            String state = null;
```

Comment:
at some point maybe we should look at the multiple calls to AuthenticationRequest.parse(authRequest)